xmm-pysas-intro-short.ipynb

```diff
@@ -0,0 +1,212 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "e10d847e-8f3d-4ad1-b0d4-c2427a7a7002",
+   "metadata": {},
+   "source": [
+    "# (Short) XMM-Newton pySAS Introduction\n",
+    "---\n",
+    "This tutorial provides a short, basic introduction to using pySAS on SciServer. A much more comprehensive introduction can be found in the <a href=\"./xmm-pysas-intro-long.ipynb\">Long pySAS Introduction</a> In this tutorial we will demonstrate,\n",
+    "\n",
+    "1. How to select a directory for data and analysis.\n",
+    "2. How to copy XMM data from the HEASARC archive.\n",
+    "3. How to run the standard XMM SAS commands `cfibuild` and `odfingest`.\n",
+    "\n",
+    "This tutorial uses an observation of NGC3079 (`obsid = '0802710101'`).\n",
+    "\n",
+    "The following assumes this notebook is run from the (xmmsas) environment on SciServer. You should see (`xmmsas`) at the top right of the notebook. If not, click there and select (`xmmsas`).\n",
```

Comment:
Can you add a comment that this will be using `gofpysas`, which is an updated? modified? version for pysas, and give some description of the changes.

What would change if the user is using the standard `pysas`. What will be different, what will they need done differently?